db/DB.js

```diff
@@ -23,6 +23,14 @@ const log = mozlog("DB");
 
 
 const DB = {
+
+  // Hashes are stored in the db using lowercase letters
+  getLowerCaseHash(email) {
+    email = getSha1(email);
+    return email.toLowerCase();
```

Comment:
Same comment as below. It feels odd that `getSha1()` returns the uppercased version of the hash (see snippet below), and then we seem to be calling `.toLowerCase()` again in most places and flipping the hash strings.

```js
  // Hashes must be uppercased to return correct results from HIBP
  return sha1EmailHash.toUpperCase();
```

Reply:
Completely agree. I'm closing this in favor of #1498 - which is (I think) a saner approach.